src/remotestorage.js

```diff
@@ -707,14 +707,17 @@ RemoteStorage.prototype = {
    * @returns {Promise} A Promise which resolves when the sync has finished
    */
   startSync: function () {
-    if (!config.cache) { return; }
+    if (!config.cache) {
+      console.warn('Can not start syncing, because caching is disabled.');
```

Comment:
I'd say "Nothing to sync" instead, but no strong opinion. If we leave it like this, grammatically more appropriate would be "cannot" without a space.